README.rst

```diff
@@ -167,7 +167,7 @@ factory_boy supports several different build strategies: build, create, and stub
     obj = UserFactory.stub()
 
 
-You can use the Factory class as a shortcut for the default build strategy:
```

Comment:
Replacing build by create does not help clarity in this context IMO. Line 155 explains that:

> factory_boy supports several different build strategies: build, create, and stub

A build strategy is the strategy used to make an object. I think we should unify the terminology, if possible on a word that is not also a strategy. Perhaps  “instantiation strategy”?

Reply:
I'd rather go for `generation` strategy then:
- Instantiation is literally instantiating the class;
- The code already provides a set of endpoints for which the user can choose the strategy, and they are named `generate`:
  - https://factoryboy.readthedocs.io/en/latest/reference.html#factory.Factory.generate
  - https://factoryboy.readthedocs.io/en/latest/reference.html#factory.generate

Reply:
Yes, that’s better.